src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -427,5 +461,29 @@ private async Task AssertAutomaticReviewModifier(ClaimsPrincipal user, ReviewMod
                 throw new AuthorizationFailedException();
             }
         }
+
+        private async Task<ReviewRevisionModel> FindMatchingApprovedRevision(ReviewModel review)
+        {
+            var revisionModel = review.Revisions.Last();
+            var revisionFile = revisionModel.Files.FirstOrDefault();
+            var codeFile = await _codeFileRepository.GetCodeFileAsync(revisionModel);
+
+            // Get automatic reviews and mark them as approved if it matches current one
```

Comment:
Is this comment incorrect? It says `automatic reviews` but the parameter is `false`